test/integration/PciManagerTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace oat\qtiItemPci\test\integration;
+
+use oat\qtiItemPci\controller\PciManager;
+use oat\qtiItemPci\model\IMSPciModel;
+use oat\qtiItemPci\model\ItemsScannerService;
+use oat\qtiItemPci\model\PciModel;
+use oat\qtiItemPci\model\portableElement\dataObject\PciDataObject;
+use oat\qtiItemPci\model\portableElement\storage\PciRegistry;
+use oat\generis\test\GenerisTestCase;
+use oat\taoQtiItem\model\portableElement\model\PortableElementModel;
+use oat\taoQtiItem\model\portableElement\PortableElementService;
+use oat\taoQtiItem\model\portableElement\storage\PortableElementRegistry;
+use \PHPUnit_Framework_MockObject_MockObject as Mock;
+use oat\taoQtiItem\model\qti\ImportService;
+
+class PciManagerTest extends GenerisTestCase
+{
+    const PRODUCT_NAME = 'TAO';
+
+    /**
+     * @var \Request|Mock
+     */
+    private $requestMock;
+    private $ontologyMock;
+    /**
+     * @var \taoItems_models_classes_ItemsService
+     */
+    private $itemsService;
+
+    /**
+     * @var PortableElementService
+     */
+    private $portableElementService;
+
+    /**
+     * @var \taoTests_models_classes_TestsService
+     */
+    private $testService;
+
+    /**
+     * @var ImportService
+     */
+    private $importService;
+
+    /**
+     * @throws \common_ext_ExtensionException
+     */
+    public function setUp()
+    {
+        $this->ontologyMock = $this->getOntologyMock();
+        $this->requestMock = $this->createMock(\Request::class);
+        $this->itemsService = \taoItems_models_classes_ItemsService::singleton();
+        $this->itemsService->setModel($this->ontologyMock);
+        $this->portableElementService = new PortableElementService();
+        $this->testService = \taoTests_models_classes_TestsService::singleton();
+        $this->importService = ImportService::singleton();
+        \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiItem');
+    }
+
+    /**
+     * @test
+     * @expectedException \oat\taoQtiItem\model\portableElement\exception\PortableElementException
+     */
+    public function unregisterRequestWithoutParameterTest()
```

Comment:
are those tests run ? (method naming does not seem to match phpunit method naming convention, start with `test...`)

Reply:
I used test annotation

Reply:
ok